numerics/elliptic_integrals.cpp

```diff
@@ -1714,137 +1699,144 @@ void FukushimaEllipticBDJ(Angle const& φ,
-  Angle e{uninitialized};
-  Angle f{uninitialized};
-  Angle ᴨ{uninitialized};
-  EllipticEFΠ(φ, /*n=*/1.0, mc, e, f, ᴨ);
-  return e;
+Angle EllipticF(Angle const& φ, double const mc) {
+  Angle F{uninitialized};
+  Angle E{uninitialized};
+  Angle Π{uninitialized};
+  // TODO(egg): we are needlessly computing J (and Π) here, which is why we have
+  // to pass n.
+  EllipticFEΠ(φ, /*n=*/1, mc, F, E, Π);
+  return F;
 }
 
-Angle EllipticF(Angle const& φ, double const mc) {
-  Angle e{uninitialized};
-  Angle f{uninitialized};
-  Angle ᴨ{uninitialized};
-  EllipticEFΠ(φ, /*n=*/1.0, mc, e, f, ᴨ);
-  return f;
+Angle EllipticE(Angle const& φ, double const mc) {
+  Angle F{uninitialized};
+  Angle E{uninitialized};
+  Angle Π{uninitialized};
+  // TODO(egg): we are needlessly computing J (and Π) here, which is why we have
+  // to pass n.
+  EllipticFEΠ(φ, /*n=*/1, mc, F, E, Π);
+  return E;
 }
 
 Angle EllipticΠ(Angle const& φ, double const n, double const mc) {
-  Angle e{uninitialized};
-  Angle f{uninitialized};
-  Angle ᴨ{uninitialized};
-  EllipticEFΠ(φ, n, mc, e, f, ᴨ);
-  return ᴨ;
+  Angle F{uninitialized};
+  Angle E{uninitialized};
+  Angle Π{uninitialized};
+  EllipticFEΠ(φ, n, mc, F, E, Π);
+  return Π;
 }
 
-void EllipticEFΠ(Angle const& φ,
+void EllipticFEΠ(Angle const& φ,
                  double const n,
                  double const mc,
-                 Angle& e,
-                 Angle& f,
-                 Angle& ᴨ) {
+                 Angle& F_φǀm,
+                 Angle& E_φǀm,
+                 Angle& Π_φ_nǀm) {
   DCHECK_LE(0, n);
   DCHECK_GE(1, n);
   DCHECK_LE(0, mc);
   DCHECK_GE(1, mc);
-  Angle b{uninitialized};
-  Angle d{uninitialized};
-  Angle j{uninitialized};
-  FukushimaEllipticBDJ(φ, n, mc, b, d, j);
-  e = b + mc * d;
-  f = b + d;
-  ᴨ = f + n * j;
+  Angle B{uninitialized};
+  Angle D{uninitialized};
+  Angle J{uninitialized};
+  FukushimaEllipticBDJ(φ, n, mc, B, D, J);
+  F_φǀm = B + D;
+  E_φǀm = B + mc * D;
+  Π_φ_nǀm = F_φǀm + n * J;
 }
 
-//  Double precision complete elliptic integral of the first kind
-//
-//     Reference: T. Fukushima, (2009) Celest. Mech. Dyn. Astron. 105, 305-328
-//        "Fast Computation of Complete Elliptic Integrlals and Jacobian
-//         Elliptic Functions"
-//
-//     Author: T. Fukushima Toshio.Fukushima@nao.ac.jp
-//
-//     Inputs: mc   = complementary parameter 0 <= mc   <= 1
-//
-Angle EllipticK(double const mc) {
-  DCHECK_LE(0, mc);
-  DCHECK_GE(1, mc);
+// Note that the identifiers in the function definition are not the same as
+// those in the function declaration.
+// The notation here follows [Fuk09], whereas the notation in the function
+// declaration uses |mc| for consistency with the other functions.
+Angle EllipticK(double const mʹ) {
+  DCHECK_LE(0, mʹ);
+  DCHECK_GE(1, mʹ);
   // TODO(phl): Use a binary split of [0, 1] to reduce the number of
   // comparisons.
-  double const m = 1.0 - mc;
+  double const m = 1.0 - mʹ;
```

Comment:
I don't think it's useful to swtch from `mc` to `mʹ` for this function.  As I recall this is the only place where Fukushima uses this notation, and the notation is in general messy and inconsistent enough that we shouldn't add to the mess.

Reply:
We mention K′ below (in the existing comment as well), so we use the notation from [Fuk09] here. If we start mixing notations between papers we are going to get a mess: calling K′ Kc would give its c a different semantic than in Bc.